scripts/travis-screenshots

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+# Decrypt the deployment key
+openssl aes-256-cbc -K $encrypted_1645300b04d0_key -iv $encrypted_1645300b04d0_iv -in .travis.sskey.enc -out .travis.sskey -d
```

Comment:
What are $encrypted_1645300b04d0_key and $encrypted_1645300b04d0_iv ? Travis environment variables? Why so starnge names? And where does .travis.sskey.enc come from?